wandb/__init__.py

```diff
@@ -24,155 +24,162 @@
 # Used with pypi checks and other messages related to pip
 _wandb_module = 'wandb'
 
+import os
 import sys
 
-from wandb.errors import Error
 
-# This needs to be early as other modules call it.
-from wandb.errors.term import termlog, termerror, termwarn
-
-PY3 = sys.version_info.major == 3 and sys.version_info.minor >= 6
-if PY3:
-    TYPE_CHECKING = True
-    from wandb import sdk as wandb_sdk
+if os.environ.get("WANDB_MODE") == "noop":  # for magic
```

Comment:
Lets not mangle __init__.py since you need to handle the case where mode is passed to wandb_init() anyways.